sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/compiler/TableInterfaceGenerator.kt

```diff
@@ -68,28 +72,37 @@ internal class TableInterfaceGenerator(private val table: SqliteCreateTableStmt)
         .addModifiers(DATA)
         .addSuperinterface(ClassName(table.sqFile().packageName, typeName))
 
-    var propertyPrints = listOf<String>()
```

Comment:
Wouldn't changing this to `listOf<CodeBlock>()` allow retaining the ability to reference a `MemberName` and retain the old single-string output?

Reply:
It would, and it does. The problem is formatting, which is what KotlinPoet still struggles with. The best I could get is something like this:
```kotlin
Test [
    foo=$foo
    bar=$bar
   ]
```
Using a `StringBuilder` provides more control over this at the cost of verbosity. Besides, looks like `appendln` is only declared in the JVM version of stdlib. I'll see if I can do better.